src/eval/operation.rs

```diff
@@ -594,7 +594,7 @@ impl<R: ImportResolver, C: Cache> VirtualMachine<R, C> {
                         let mut shared_env = Environment::new();
                         let f_as_var = f.body.closurize(&mut self.cache, &mut env, f.env);
 
-                        // Array elements are closurized to preserve lazyness of data structures. It
+                        // Array elements are closurized to preserve laziness of data structures. It
```

Comment:
Ah, I've always thought lazyness was a valid US spelling. But it seems it's actually not the case.

Reply:
So no change on my side here ...